src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/javadoc/writetag/InputWriteTagRecordsAndCompactCtors.java

```diff
@@ -21,45 +21,45 @@ public class InputWriteTagRecordsAndCompactCtors { // violation
     class TestClass {
     }
 
-    // violation 2 lines below , 'Javadoc tag .*'
+    // violation 2 lines below 'Failed to recognize 'record' introduced in Java 14.'
     /**
-     * @incomplete // violation
+     * @incomplete Failed to recognize 'record' introduced in Java 14.
      */
     record MyRecord1() {
 
     }
 
-    // violation below , 'Type Javadoc comment is missing @incomplete tag'
-    record MyRecord2(String myString) { // violation
 
-        // violation 2 lines below , 'Javadoc tag .*'
+    record MyRecord2(String myString) { // violation 'missing @incomplete tag.'
+
+        // violation 2 lines below 'Failed to recognize 'record' introduced in Java 14.'
         /**
-         * @incomplete // violation
+         * @incomplete Failed to recognize 'record' introduced in Java 14.
          */
         public MyRecord2 {
         }
     }
 
-    // violation below , 'Type Javadoc comment is missing @incomplete tag'
-    record MyRecord3(int x) { // violation
 
-        // violation 2 lines below , 'Javadoc tag .*'
+    record MyRecord3(int x) { // violation 'Type Javadoc comment is missing @incomplete tag.*'
+
+        // violation 2 lines below 'Failed to recognize 'record' introduced in Java 14.'
         /**
-         * @incomplete // violation
+         * @incomplete Failed to recognize 'record' introduced in Java 14.
```

Comment:
it sounds like a defect , right ?
if you think so, please created issue on this 

.... and you can fix it as your next issue, as you might be already so familiar with this Check, it should be easy win.

Reply:
@Sloent , please do this

Reply:
First of all, thank you very much for your constructive suggestions during this process.  Moreover, this is also my first attempt to deal with your new question.  Please allow me to ask the following questions:

①Are you saying that you want me to make further changes so that java versions 14 and up can pass the test?

②Do issues have to be submitted simultaneously with PR?

③If I want to make sure I'm on the right track, or if there's anything I need to add, where should I discuss it?

Reply:
Please create new  issue in our issue tracker.
I will approve it, and you can send PR to fix it.
Or maybe send PR after you fix gsoc third issues.

Reply:
@Sloent , please create issue.

Reply:
Ok, I'll bring up the issue tonight

Reply:
I have created it, please approve

Reply:
```
// violation 2 lines below 'Failed to recognize 'record' introduced in Java 14.'
    /**
     * @incomplete Failed to recognize 'record' introduced in Java 14.
     */
    record MyRecord1() {

    }
```
The reason for the error in this code is that the 'record' keyword is an illegal identifier in Java 11-13. Therefore, when the compiler encounters the 'record' keyword, it throws an 'illegal start of expression' compilation error.

In this case, the Java version that compiles 'record' must be version 14 or later. This way the following code will not report an error:

```
/**
 * @incomplete This class needs more code...
 */
 record MyRecord1() {
 
 }
```

Therefore, in order to fill in the gaps, I think it is OK to make the test cases that comply with the rule "' @incomplete 'must be followed by a non-empty description" legal.